examples/sinh_libtorch/README.md

```diff
@@ -3,8 +3,9 @@
 ```
 mkdir build
 cd build
-cmake -DCMAKE_PREFIX_PATH="$(python -c 'import nvfuser.utils; import torch.utils; print(nvfuser.utils.cmake_prefix_path, torch.utils.cmake_prefix_path, sep=";")')" ..
-make -j
+CMAKE_PREFIX_PATH=$(python -c 'import nvfuser.utils; import torch.utils; print(nvfuser.utils.cmake_prefix_path, torch.utils.cmake_prefix_path, sep=";")')
+cmake -DCMAKE_PREFIX_PATH=$CMAKE_PREFIX_PATH -G Ninja ..
```

Comment:
Changing this to use ninja by default. make is still supported, but make will not tell me the command to compile the source file, so it is very inconvenient for debugging. Instead, ninja will print something very nicely helpful:

```C++
❯ cmake --build .
[1/2] Building CXX object CMakeFiles/sinh_example.dir/main.cpp.o
FAILED: CMakeFiles/sinh_example.dir/main.cpp.o 
/usr/bin/c++ -DUSE_C10D_GLOO -DUSE_C10D_MPI -DUSE_C10D_NCCL -DUSE_DISTRIBUTED -DUSE_RPC -DUSE_TENSORPIPE -isystem /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/torch/include -isystem /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/torch/include/torch/csrc/api/include -isystem /usr/local/cuda/include -isystem /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser -std=gnu++17 -D_GLIBCXX_USE_CXX11_ABI=1 -MD -MT CMakeFiles/sinh_example.dir/main.cpp.o -MF CMakeFiles/sinh_example.dir/main.cpp.o.d -o CMakeFiles/sinh_example.dir/main.cpp.o -c /home/gaoxiang/Fuser3/examples/sinh_libtorch/main.cpp
In file included from /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/ir/base_nodes.h:15,
                 from /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/ir/all_nodes.h:10,
                 from /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/device_lower/analysis/trivial_broadcast.h:10,
                 from /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/compute_at_map.h:10,
                 from /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/device_lower/lower2device.h:12,
                 from /home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/executor.h:9,
                 from /home/gaoxiang/Fuser3/examples/sinh_libtorch/main.cpp:1:
/home/gaoxiang/.virtualenvs/nvfuser/lib/python3.11/site-packages/nvfuser-0.0.19+git5dd5fac-py3.11-linux-x86_64.egg/nvfuser/include/nvfuser/polymorphic_value.h:22:10: fatal error: dynamic_type.h: No such file or directory
   22 | #include <dynamic_type.h>
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.

```

Reply:
make can also be verbose :shrug:

Reply:
> make can also be verbose 🤷

Haha, thank you for mentioning that. I have no idea on how customize make or ninja. I just know that the default behavior of ninja seems to make more sense than make.